public/video-ui/src/services/pandaReqwest.js

```diff
@@ -8,24 +8,24 @@ function poll(reqwestBody, timeout) {
 
   function makeRequest(resolve, reject) {
     reqwest(reqwestBody).then(response => resolve(response)).fail(err => {
-      if (err.status === 419) {
-        const store = getStore();
-        const reauthUrl = store.getState().config.reauthUrl;
-
-        reEstablishSession(reauthUrl, 5000).then(
-          () => {
-            setTimeout(makeRequest, interval, resolve, reject);
-          },
-          error => {
-            throw error;
-          }
-        );
-      } else {
-        if (Number(new Date()) < endTime) {
-          setTimeout(makeRequest, interval, resolve, reject);
+      if (Number(new Date()) < endTime) {
```

Comment:
any reason for this change? linter?

Reply:
Ah should've mentioned it in the PR comment. It stops retrying forever if the response is a 419 (like not being logged into Workflow)